YCAI/src/components/dashboard/LinkAccount.tsx

```diff
@@ -124,12 +136,12 @@ export const LinkAccount: React.FC<LinkAccountProps> = ({ auth }) => {
 
   const handleGoBackToStepOneClicked = (): void => {
     setVerificationFailed(false);
+    setSubmitChannelFailed(false);
+    setChannel('');
```

Comment:
are we triggering 3 renders here? can be better to have one single state as object?

Reply:
Apparently not, because the setStates are within an event handler and thus are pooled (https://medium.com/@guptagaruda/react-hooks-understanding-component-re-renders-9708ddee9928). I've confirmed this empirically with a consol.log() during the render: there are 4 renders with the 3 setState, and there are still 4 renders with just one setState. Why we have 4 I do not know, but that doesn't seem to be due to the setStates.